gulpfile.js

```diff
@@ -168,9 +168,9 @@ gulp.task('injectscripts', function() {
 });
 
 gulp.task('background', function() {
-  return browserify('src/background/main.js')
+  return browserify('src/background.js')
```

Comment:
The idea here was to create seperate files in this directory for each task that all get bundled together into 1, but this is fine for now.